packages/measure-tools/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "@bentley/measure-tools-react",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Frontend framework and tools for measurements",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/measure-tools"
+  },
+  "main": "lib/measure-tools.js",
+  "typings": "lib/measure-tools",
+  "scripts": {
+    "build": "npm run copy:assets && npm run pseudolocalize && tsc 1>&2 && npm run webpack:tests",
+    "copy:assets": "cpx \"./src/**/*.*css\" \"./lib\" && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "watch": "tsc --w",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "lint": "tslint --project . 1>&2",
```

Comment:
Any reason to not update to use eslint? Maybe we just take a separate pass through the whole repo...

Reply:
I thought of that, there were some other pkgs using tslint, and thought itd be best to do it all at once.